jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/editor/CodeWhispererEditorListener.kt

```diff
@@ -18,6 +23,12 @@ class CodeWhispererEditorListener : EditorFactoryListener {
             object : DocumentListener {
                 override fun documentChanged(event: DocumentEvent) {
                     CodeWhispererInvocationStatus.getInstance().documentChanged()
+                    editor.project?.let { project ->
+                        PsiDocumentManager.getInstance(project).getPsiFile(editor.document)?.toProgrammingLanguage() ?. let { languageName ->
+                            val language = ProgrammingLanguage(languageName)
```

Comment:
nit: just `language = ProgrammingLanguage(languageName).toCodeWhispererLanguage()` to be cleaner?

Reply:
ack